docs/dev-guide/integration/rest-api/server_plugins.rst

```diff
@@ -0,0 +1,336 @@
+Server's Plugins API
+=====================
+
+The pulp server itself has very little built in functionality, so it loads number of plugins
+in order to provide desired functionality required by administrators.
```

Comment:
The pulp server itself has very little built in functionality, so loads plugins that provide the desired functionality.